### ISSUE TITLE
"Live mode" to control the features 

### ISSUE THREAD

**Issue by Gajenthran** [FrancescoCeruti/linux-show-player#155]
Hello,
I would like to know if adding a "live mode" in LISP is a good idea. The live mode will lock certain features which will not be useful for the mode. For example, the live mode will lock editing or interrupting cues.

**Comment by fralix** [comment]
Yeah, this would be a nice feature. It's also present in other show software, like QLC+ for example.

This would kinda introduce a use to the GO button to me :rofl: 

Please make sure not to introduce workflow caveeats by switching between the modes (like stopping running cues etc).

It might seem a good idea to prevent accidental cue starting while in edition mode, but I would probably be limited by this protection, as I sometimes have to edit things while running the show. 

A three-mode system would solve this issue: Preparation, Live and Preview (which is the current behaviour, I just came up with the while writing the next part, I still don't know if it suits it's purpose).
Preparation locks accidently launching cues, Preview is the current behaviour with editing and running, and live is running only.

**Comment by FrancescoCeruti** [comment]
I think that in the formula suggested by @fralix it could be a very good option, this way anyone can use it's preferred workflow.

**Comment by leonerd** [comment]
Yes - I would find a "Live Show" mode very useful to lock out all the editing, putting it into a much simpler mode for operating the show for real, as compared to editing it.
